### PR TITLE
[2.0.5] en: add config command for v2.0.5

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/736
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/maintain-dm-using-tiup.md [hunk 1]
```diff
@@ -177,6 +177,16 @@ For example, to scale out a DM-worker node in the `prod-cluster` cluster, take t
 
 ## Rolling upgrade
 
+> **Note:**
+>
+> Since v2.0.5, dmctl support [Export and Import Data Sources and Task Configuration of Clusters](export-import-config.md)。
+>
+> Before upgrading, you can use `config export` to export the configuration files of clusters. After upgrading, if you need to downgrade to an earlier version, you can first rebuild the earlier cluster and then use `config import` to import the previous configuration files.
```
Comment (GMHDBJD) [inline en/maintain-dm-using-tiup.md:184]:
```suggestion
> Before upgrading, you can use `config export` to export the configuration files of clusters. After upgrading, if you need to downgrade to an earlier version, you can first redeploy the earlier cluster and then use `config import` to import the previous configuration files.
```

**Review by GMHDBJD — COMMENTED** [review]
rest LGTM

File: en/export-import-config.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: Export and Import Data Sources and Task Configuration of Clusters
+summary: Learn about the configuration of exporting and importing data sources and task configuration of clusters when you use DM.
```
[the remark this answers is not in the document]
Comment (Liuxiaozhen12) [reply en/export-import-config.md]: solved in https://github.com/pingcap/docs-dm/pull/743/commits/e9bba5067c9b610c02fec743c92bd2483cd231c6.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by lance6716 — APPROVED** [review]

**Pull request merged**